modules/auxiliary/gather/ldap_esc_vulnerable_cert_finder.rb

```diff
@@ -91,6 +99,7 @@ def initialize(info = {})
   CERTIFICATE_ENROLLMENT_EXTENDED_RIGHT = '0e10c968-78fb-11d2-90d4-00c04f79dc55'.freeze
   CERTIFICATE_AUTOENROLLMENT_EXTENDED_RIGHT = 'a05b8cc2-17bc-4802-a710-e7c15ab866a2'.freeze
   CONTROL_ACCESS = 0x00000100
+  CT_FLAG_NO_SECURITY_EXTENSION = 0x80000
```

Comment:
Based on the value and the [section](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-crtd/ec71fd43-61c2-407b-83c9-b52272dec8a1) in which it's defined, it looks like this constant should be moved to this [line here.](https://github.com/rapid7/metasploit-framework/blob/5e2418cf1f68a392981b832e72d1731f322d0ebb/lib/rex/proto/ms_crtd.rb#L39)